scripts/deployment-test/package.json

```diff
@@ -9,9 +9,9 @@
     "@cloudflare/wrangler": "latest",
     "@iarna/toml": "^2.2.5",
     "@octokit/rest": "^18.12.0",
+    "@remix-run/dev": "1.3.0-pre.1",
```

Comment:
Can you please make a note to come back in and update this once we publish 1.3? Thanks!

Reply:
will do, but it should be updated by the `version` script :)